### PR TITLE
copy host network feature page: web-admin usage details

### PR DESCRIPTION
**Pull request by erav**
Changes proposed in this pull request:

- add image and elaborate on explanation for usage via web-admin

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

**Comment by erav** [comment]
reviewer:
@almusil 
@dominikholler

File: source/develop/release-management/features/network/copy-host-networking.html.md [hunk 1]
```diff
@@ -43,17 +43,23 @@ This method takes a single parameter, the host ID from which to copy the configu
...
-6. Click the `Copy` button. When the destination host finishes copying the network configuration, it reboots with
-the new network configuration.
+4. When the source host is selected in the hosts list, click the `Copy Host Networks` button or right-click the host
+   and select the `Copy Host Networks` option from the context menu.
+   * Note: only one host must be selected for this action to be available.
+5. In the popup window that appears, select the destination host to which the configuration is to be copied.
+   * Note: only hosts in the same cluster as the source host will be listed as potential destinations.
+6. Click the `Copy Host Networks` button. 
+7. Engine applies the configuration and posts a notification when done.
+8. When the destination host finishes copying the network configuration, it reboots with the new network configuration.
```
Comment (dominikholler) [inline source/develop/release-management/features/network/copy-host-networking.html.md:60]: I cannot remember why we have the sentence
`When the destination host finishes copying the network configuration, it reboots with the new network configuration.`
Maybe we can remove it?

Comment (erav) [reply source/develop/release-management/features/network/copy-host-networking.html.md]: done

**Comment by dominikholler** [comment]
Looks good.

**Comment by dominikholler** [comment]
@sandrobonazzola can you please merge?

**Pull request merged**